docker/conf/apache2.conf

```diff
@@ -0,0 +1,35 @@
+<Virtualhost *:80>
+    ServerName  selfservice
+    ServerAdmin admin@dev.openconext.local
+
+    DocumentRoot /var/www/html/public
+
+    SetEnv HTTPS on
+    SetEnv APP_ENV prod
```

Comment:
Can this be removed? Setting it in the Apache config makes overriding this var quite hard.

Reply:
Yes, you're absolutely right. I've removed it now!